app/helpers/pageflow/admin/users_helper.rb

```diff
@@ -4,6 +4,14 @@ module UsersHelper
       def collection_for_user_roles
         User.roles_accessible_by(current_ability).index_by { |role| t(role, :scope => 'pageflow.admin.users.roles') }
       end
+
+      def delete_me_para
+        if Pageflow.config.authorize_user_deletion.call(current_user) == true
+          render('may_delete')
+        else
+          render('cannot_delete')
+        end
```

Comment:
* Let's save the result of `call` here and pass it to the partial as local variable. That way we do not call the lambda multiple times and do not risk calling it without parameters in some cases (see below)
* To catch such errors, let's add a test which covers the two cases and makes sure the returned string is rendered.
* Since all helpers share a global scope, helper method names should include the name of their module, i.e. `delete_own_user_section` etc.